### PR TITLE
overlap songs above tiles closes #42

### LINKED ISSUE #42: Album songs should overlap the album tiles on hover
In mobile and Ipad view, albums names are currently hidden behind tiles. They should be above the tiles.

### PR DESCRIPTION
**Pull request by sourabhpruthi10**
* Please fill out the template below

### Did you read the instructions in [README](../README.md) ?

>Yes
### Solves which issue ?

>Solves #42
### Give discription.

>[Write Here. Be Specific]

### CODE REVIEW

**Comment by pradeepgangwar** [comment]
@sourabh1031 Please provide a live link to review changes.

**Comment by sourabhpruthi10** [comment]
https://sourabh1031.github.io/Selena-Gomez-Discography/

File: css/theme.css [hunk 1]
```diff
@@ -129,4 +128,7 @@ body {
     height: 40vw;
     width: 63vw
     }
+    .dropbtn{
+        z-index: -1;
```
Comment (OrionStar25) [inline css/theme.css]: I think `z-index` should be positive. Right now, album tiles are overlapping the songs. We need the opposite.

**Review by OrionStar25 — CHANGES_REQUESTED** [review]
You have merge conflicts. Please try to remove them.

**Comment by OrionStar25** [comment]
Still not working for mobile view. Please check if the commit works on your local link before committing.

**Comment by sourabhpruthi10** [comment]
@OrionStar25 i think now it works.

**Comment by OrionStar25** [comment]
Still not working. 
screenshot from 2019-01-24 22-37-32

**Pull request closed without merging**